cobbler/items/item.py

```diff
@@ -171,8 +171,8 @@ def _resolve(self, property_name: str) -> Any:
         settings_name = property_name
         if property_name.startswith("proxy_url_"):
             property_name = "proxy"
-        if property_name == "default_ownership":
-            property_name = "owners"
+        if property_name == "owners":
+            settings_name = "default_ownership"
```

Comment:
Is `settings_name` really expected here?

Reply:
Yes because we do want to fall through to the lines 190 and 191 to access the settings in case we have no other parent. In case we have a parent we do want to stick to the field name because it is identical.